File: VariablesConstantsbucle/fase2.js
var refYear = 1948
var leapYear = []
var myBirthday = parseInt(prompt("your birth year?"))

for (i = refYear + 1; i <= myBirthday; i++)
    if ((i - refYear) % 4 == 0) {
        leapYear.push(i)
    }
alert("there are " + leapYear.length + " between " + refYear + " and now: \n" + leapYear)

File: VariablesConstantsbucle/Fase2.java
public class Fase2 {

	public static void main(String[] args) {
		final int RefYear = 1948;
		int interval = 4;
		int MyYear = 1978;
		int numberOfLeapYears = (MyYear - RefYear) / 2;

		System.out.println("there are " + numberOfLeapYears + " leap years between " + RefYear + " and " + MyYear);

	}

}
